web/src/components/tasks/business-formation/business/FormationDate.tsx

```diff
@@ -73,30 +75,36 @@ export const FormationDate = (props: Props): ReactElement => {
     });
   };
 
+  const getMinDate = (): dayjs.Dayjs | undefined => {
+    const startDate = dayjs(getCurrentDateInNewJersey().format(dateFormat));
+    return props.fieldName === "businessStartDate" ? startDate : undefined;
+  };
+
```

Comment:
There should be a way to set a relative locale via the `LocalizationProvider`, but everything I tried wasn't working. What did work was getting the current date in NJ, creating a string, and then creating a new date object to be used in setting the date minimum. I don't love this, but it works.